drush/sites/archaeology.site.yml

```diff
@@ -6,6 +6,11 @@ dev:
   uri: archaeology-dev.stanford.edu
   host: srv-7503.devcloud.hosting.acquia.com
   user: swshumsci.dev
+livedev:
```

Comment:
Do you need this? Doesn't drush hit the same db even if in livedev? I guess if you are creating a drush plugin this might be important.

Reply:
I thought I needed it because of the settings that acquia was adjusting. Probably don't now that I think about it

Reply:
it does kinda make `drush ssh` a little faster too

Reply:
Nice to have. Just typing out loud.